big_tests/src/ct_markdown_errors_hook.erl

```diff
@@ -161,3 +168,24 @@ reindent(Bin) ->
     %% Use 2 whitespaces instead of 4 for indention
     %% to make more compact look
     binary:replace(Bin, <<"    ">>, <<"  ">>, [global]).
+
+
+get_group_names(Config) ->
+    %% tc_group_path contains something like:
+    %% [[{name,muc_rsm_all},parallel],
+    %%  [{name,rdbms_muc_all},{repeat_until_all_ok,3},parallel]]
+    %% Where muc_rsm_all is subgroup of rdbms_muc_all.
+    Path = proplists:get_value(tc_group_path, Config, []),
+    TopGroup = proplists:get_value(tc_group_properties, Config, []),
+    Names = [ proplists:get_value(name, Props) || Props <- lists:reverse([TopGroup|Path]) ],
```

Comment:
Cool, this is very cool, I can confirm I've had info missing while fighting with mam super-nested grouping. I want to see what report this generates, I want to see this working and I'll approooooveeeee

Reply:
Here https://github.com/arcusfelis/MongooseIM/pull/50#issuecomment-510935761

Reply:
I love it, very cool!